CRM/Core/DAO.php

```diff
@@ -99,15 +99,17 @@ public static function getTableName() {
    *   The database connection string.
    */
   public static function init($dsn) {
+    Civi::$statics[__CLASS__]['init'] = 1;
     $options = &PEAR::getStaticProperty('DB_DataObject', 'options');
     $options['database'] = $dsn;
     if (defined('CIVICRM_DAO_DEBUG')) {
       self::DebugLevel(CIVICRM_DAO_DEBUG);
     }
     CRM_Core_DAO::setFactory(new CRM_Contact_DAO_Factory());
-    if (CRM_Utils_Constant::value('CIVICRM_MYSQL_STRICT', CRM_Utils_System::isDevelopment())) {
-      CRM_Core_DAO::executeQuery('SET SESSION sql_mode = STRICT_TRANS_TABLES');
+    if (\CRM_Utils_Constant::value('CIVICRM_MYSQL_STRICT', \CRM_Utils_System::isDevelopment())) {
```

Comment:
@totten want to do a quick `git commit --amend` and `git push -f` to get rid of the supurflous changes so we can merge?